CMakeLists.txt

```diff
@@ -38,12 +33,23 @@ else()
 endif()
 
 # add SDK
```

Comment:
Why not putting those dependencies in their own cmake file, and including the cmake file here? I usually find it makes the root CMakeLists a bit easier to understand.

Reply:
Definitely!

Reply:
So I tried putting the dependencies into CMakeLists.txt  files in subdirectory which didn't work (it had troubles copying some of the stamp files and I couldn't get it to work. I'll now try to use a cmake file instead.